bundles/org.openhab.binding.ojelectronics/src/main/java/org/openhab/binding/ojelectronics/internal/config/OJElectronicsBridgeConfiguration.java

```diff
@@ -52,8 +52,17 @@ public class OJElectronicsBridgeConfiguration {
      */
     public int softwareVersion = 1060;
 
-    /**
-     * Refresh-Delay
+    /*
+     * Gets the Api-URL
+     */
+    public String getApiUrl() {
+        return apiUrl.replace("/api", "") + "/api";
```

Comment:
Can you explain this logic? `apiUrl` is private and doesn't seem changed by any method in this class? Unless I'm missing something, it could be made `final`, and this method could be simplified to:
```suggestion
        return apiUrl + "/api";
```

Reply:
The reason for the replace is that default value of the former version of this parameter ends with "/api". I decided to use this parameter for all requests to the server. Sometimes without "/api". To keep compatibility this logic replaces "/api" with nothing

Reply:
I totally missed that this is a configuration DTO, sorry.